README.md

```diff
@@ -1,65 +1,35 @@
-# vscode-openshift-tools README
+# Visual Studio Code OpenShift Connector
 
-[![Build Status](https://travis-ci.org/redhat-developer/vscode-openshift-tools.svg?branch=master)](https://travis-ci.org/redhat-developer/vscode-openshift-tools)
+[![Build Status](https://travis-ci.org/redhat-developer/vscode-openshift-tools.svg?branch=master)](https://travis-ci.org/vscode-openshift-tools)
```

Comment:
What's the rationale for this change. The second URL is invalid

Reply:
Fixed the  invalid url. That should not have been changed.